lucene/core/src/java/org/apache/lucene/codecs/lucene90/PForUtil.java

```diff
@@ -35,6 +46,7 @@ static boolean allEqual(long[] l) {
   }
 
   private final ForUtil forUtil;
+  private final byte[] exceptionBuff = new byte[14];
```

Comment:
Maybe we should extract a constant for the max number of exceptions so that we could then do something like that here, to make reading easier?
```suggestion
  private final byte[] exceptionBuff = new byte[MAX_EXCEPTIONS/2];
```

Reply:
Good suggestion. I added this constant and updated all of PForUtil to be aware of it.